.github/workflows/release-python.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow builds Python distribution packages using cibuildwheel tool and
+# environment and publishes packages as a part of a GitHub release.
+
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+
+name: release python
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-tag:
+        description: 'GitHub release tag to publish packages'
+        required: false
+        type: string
+      prerelease:
+        description: 'Flag to create a pre-release'
+        default: true
+        required: false
+        type: boolean
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-11]
+      max-parallel: 3
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Build wheels on Linux
+        if: runner.os == 'Linux'
+        uses: pypa/cibuildwheel@v2.13.1
+        env:
+          # Linux cannot call "sh -c ./python/install-hyperonc.sh" because
+          # before-all command is also called with "sh -c"
+          CIBW_BEFORE_ALL: ./python/install-hyperonc.sh -u https://github.com/${{github.repository}}.git -r ${{github.ref}}
+        with:
+          package-dir: ./python
+
+      - name: Build wheels on MacOSX
+        if: runner.os == 'macOS'
+        uses: pypa/cibuildwheel@v2.13.1
+        env:
+          # Mac OS X cannot call "./python/install-hyperonc.sh" probably because
```

Comment:
I don't think this is completely accurate.

Mac *has* a shell binary at `/bin/sh` (I'm pretty sure it's installed as part of the base OS, and not the xcode developer tools).  On Mac OS X 12.6, the version is: `GNU bash, version 3.2.57(1)-release`

However, the default shell on MacOS is `zsh 5.8.1` (again on Mac OS X 12.6.  so any environment setup and config likely won't be seen by `bash`

Reply:
Well, truth is if I use "./python/install-hyperonc.sh" as a value of CIBW_BEFORE_ALL, build fails, please see [macosx build results](https://github.com/vsbogd/hyperon-experimental/actions/runs/5471670075/jobs/9963037785)